rapid_response_xblock/static/js/src/rapid.js

```diff
@@ -43,6 +46,227 @@
       });
     }
 
+    function initD3() {
+      // This function is for creating and updating elements right after page load.
+
+      var svg = d3.select("#rapid-response-results").append("svg");
```

Comment:
I think it's better to define the selectors for elements in the template together (around line 13)